src/rpp/rpp/operators/merge.hpp

```diff
@@ -47,6 +51,8 @@ struct merge_on_error
 {
     void operator()(const std::exception_ptr& err, const constraint::subscriber auto& sub, const auto& state) const
     {
+        state->childs_subscriptions.unsubscribe();
+
```

Comment:
@victimsnino How would unsubscribing the child subscription help? I thought unsubscribing the child subscription only removes the child subscription from the parent subscription (and no unsubscribing parent).